utils/freifunk-berlin-migration/uci-defaults/freifunk-berlin-01-migration.sh

```diff
@@ -405,6 +405,20 @@ r1_0_2_notunnel_macaddr() {
   fi
 }
 
+r1_0_2_add_olsrd_garbage_collection() {
+  crontab -l | grep "rm -f /tmp/olsrd\*core"
+  if [ $? == 1 ]; then
+    log "adding garbage collection of core files from /tmp"
+    echo "23 4 * * *	rm -f /tmp/olsrd*core" >> /etc/crontabs/root
+    /etc/init.d/cron restart
```

Comment:
why not using the "reload" option ?
but as "restart" and "reload" do the same, it doesn't matter

Reply:
I followed the same style as above in the migration script.

I don't know if reload will work either, since it isn't a file in /etc/config.